core/templates/dev/head/pages/exploration_player/hint_button_directive.html

```diff
@@ -8,7 +8,7 @@
   </md-button>
 </span>
 <span ng-if="!allHintsAreExhausted()">
-  <span ng-if="currentHintIsAvailable()" tooltip="<['I18N_PLAYER_HINT_IS_AVAILABLE' | translate]>" tooltip-placement="left">
+  <span ng-if="currentHintIsAvailable()" tooltip-is-open="isShowingNeedHintMessage" tooltip="<[isShowingNeedHintMessageText ? ('I18N_PLAYER_HINT_NEED_A_HINT' | translate) : ('I18N_PLAYER_HINT_IS_AVAILABLE' | translate)]>" tooltip-placement="left" ng-mouseover="hideNeedHintMessage()">
```

Comment:
We might need to make this change throughout the app wherever tooltips are used. 
` tooltip-popup is now deprecated. Use uib-tooltip-popup instead.` shows in the console currently for tooltips.

Reply:
Oh strange I wasn't seeing those, but I'll change it